flags.gn

```diff
@@ -1,6 +1,6 @@
 clang_use_chrome_plugins=false
-closure_compile=false
 enable_hangout_services_extension=false
+enable_js_type_check=false
```

Comment:
`enable_js_type_check=true` relies on the pruned closure compiler binary. Since we had its predecessor (`closure-compile`) disabled, I didn't bother working on a way to replace the pruned binary.

Reply:
Not only in requires this binary, it also requires java. I also have this flag disabled by default for anyone, not willing to pull in java in dependencies.

Reply:
I see, thanks for clarifying.